changelog/cpp98.dd

```diff
@@ -0,0 +1,57 @@
+## Transition to C++11 character types
+
+        With C++11 comes the advent of changed character type mangling.
+        D's default behavior is now to conform to this. However, this
+        will break existing code. Therefore, the old behavior can be
+        retained by using the -version=Cpp98 compiler switch.
+        For Win32 compilations using Digital Mars C++, -version=Cpp98 will
+        be predefined.
+
+        Of particular note is the new difference between wchar and wchar_t on
+        Windows. This will manifest itself as compile errors when
+        interfacing wchar* with wchar_t* code when calling the Windows API.
+        A cast will resolve the issue.
+
+        Going forward we recommend using WCHAR instead of wchar or wchar_t
+        when interfacing to Windows API functions. (WCHAR is Microsoft
+        Windows' 16 bit character type.)
+
+### C++ Type Equivalence
+
+#### Cpp98 behavior:
+
+        D       Posix           DMC++ Windows   VC++ Windows
+
+        wchar   unsigned short  wchar_t         wchar_t
+        dchar   wchar_t         unsigned        unsigned
+        wchar_t wchar_t         wchar_t         wchar_t
+        WCHAR   --              wchar_t         wchar_t
+
+#### New behavior:
+
+        D       Posix           DMC++ Windows   VC++ Windows
+
+        wchar   char16_t        wchar_t         char16_t
+        dchar   char32_t        unsigned        char32_t
+        wchar_t wchar_t         wchar           wchar_t
+        WCHAR   --              wchar           wchar_t
```

Comment:
What is `wchar` in C++? There's no C++ type called `wchar` that I'm aware of... Is it just a DMC thing?
Otherwise, this looks correct to me. And I completely endorse using WCHAR in Win32 API's.